### ISSUE TITLE
Mid-project release preparation

### ISSUE THREAD

**Issue by iyadh-97** [amosproj/amos2021ws04-auto-panelist-detection#35]
## User story
1. As a team member,
2. I need to prepare the mid-project release,
3. So that we successfully perform the mid-project release. 

## Acceptance criteria
* DoDs of project release are met
* Mid-project release plan is done.

## Definition of done
* Added only after week 5
* The same for all features

**Comment by iyadh-97** [comment]
Already done.
